accounts/cachedb_test.go

```diff
@@ -232,6 +233,7 @@ func TestCacheAddDeleteOrder_CacheDB(t *testing.T) {
 	if cache.hasAddress(wantAccounts[0].Address) {
 		t.Errorf("expected hasAccount(%x) to return false", wantAccounts[0].Address)
 	}
+	cache.close()
```

Comment:
I wonder if we should just make this close:
```go
defer cache.close()
```
right after we've created a new one for future debugging purposes and easy reading since we forgot to include it the previous versions.